plugins/modules/efs_info.py

```diff
@@ -387,7 +382,6 @@ def main():
         targets = [(item, prefix_to_attr(item)) for item in targets]
         file_systems_info = [item for item in file_systems_info if has_targets(item['mount_targets'], targets)]
 
-    if is_old_facts:
         module.exit_json(changed=False, ansible_facts={'efs': file_systems_info})
```

Comment:
this still appears to be returning a fact

Reply:
/me headdesk - I actually made these changes then I stashed them while testing back and forth between main and this branch.  Thank you.  :smile: